orchagent/vxlanorch.cpp

```diff
@@ -264,6 +265,17 @@ create_tunnel(
         tunnel_attrs.push_back(attr);
     }
 
+    if (encap_ttl != 0)
```

Comment:
since you're not exposing any places for user to change the `encap_ttl` value, why do you need to expose the capability of setting it via different functions while providing all the functions with default value? could you directly set the TTL mode to pipe model and the value to the default value?

Reply:
@stcheng , the VxlanTunnel is created for different scenarios. For the VNET use-case, we would want the Tunnel to have default 128 TTL. So I think it is better to pass this as parameter.